scripts/run_molecule

```diff
@@ -45,14 +46,26 @@ for rolepath in ${ROLES_LIST}; do
     test -d ${role} || continue
     echo "Running molecule against: ${role}"
     pushd $role
-    case ${USE_VENV} in
-        y|yes|true)
-            ${HOME}/test-python/bin/molecule -c "${PROJECT_DIR}${MOLECULE_CONFIG}" ${TEST_VERBOSITY} test --all
-            ;;
-        *)
-            molecule -c "${PROJECT_DIR}${MOLECULE_CONFIG}" ${TEST_VERBOSITY} test --all
-            ;;
-    esac
+    MOLECULE_RESULTS=$(\
+        case ${USE_VENV} in
+                y|yes|true)
+                    ${HOME}/test-python/bin/molecule -c "${PROJECT_DIR}${MOLECULE_CONFIG}" ${TEST_VERBOSITY} test --all
+                    ;;
+                *)
+                    molecule -c "${PROJECT_DIR}${MOLECULE_CONFIG}" ${TEST_VERBOSITY} test --all
+                    ;;
+        esac ) || true
+        if [[ $MOLECULE_RESULTS =~ failed=[1-9] ]]; then
```

Comment:
what if we get 10 failures?

Reply:
I'ts regex. As long as the first number is anything else then zero it will match. 

https://wiki.bash-hackers.org/syntax/ccmd/conditional_expression

Reply:
ah, right, no full matching. Need more coffee.